src/capi/azext_capi/custom.py

```diff
@@ -164,27 +165,42 @@ def create_management_cluster(cmd, yes=False):
                 found_cluster = True
             except subprocess.CalledProcessError as err:
                 raise UnclassifiedUserFault("Can't locate a Kubernetes cluster") from err
-    if not found_cluster and not create_new_management_cluster(cmd):
+    if not found_cluster and not create_new_management_cluster(cmd, cluster_name=cluster_name):
         return
     set_azure_identity_secret_env_vars()
     _create_azure_identity_secret(cmd)
     _install_capz_components(cmd)
 
 
-def create_new_management_cluster(cmd, prompt=True, pre_prompt_text=None):
+def get_cluster_name_by_user_prompt():
+    while True:
+        prompt_res = prompt_method("Please enter a name for the management cluster: ")
+        if re.match("^[a-z0-9.-]+$", prompt_res):
+            return prompt_res
+        msg = "Invalid name for cluster, only lowercase characters, numbers, dashes and periods allowed"
+        logger.error(msg)
+
+
+def create_new_management_cluster(cmd, prompt=True, pre_prompt_text=None, cluster_name=None):
     choices = ["azure - a management cluster in the Azure cloud",
                "local - a local Docker container-based management cluster",
                "exit - don't create a management cluster"]
 
+    default_cluster_name = "capi-manager"
     if prompt:
         prompt = pre_prompt_text if pre_prompt_text else ""
         prompt += """
 Where do you want to create a management cluster?
 """
         choice_index = prompt_choice_list(prompt, choices)
+        prompt = f"Do you want to name the management cluster? [default:{default_cluster_name}]"
```

Comment:
instead of asking the user if y/n they want to name it, what do you think about directly asking them what the name should be with the default being used if they enter nothing and directly press enter?